stdlib/Markdown/src/Markdown.jl

```diff
@@ -122,4 +122,8 @@ import Base.Docs: catdoc
 
 catdoc(md::MD...) = MD(md...)
 
+if Base.generating_output()
+    dummy = md""  # workload to trigger precompile
```

Comment:
Why are you assigning this to a global variable which gets baked into the sysimage?

Reply:
I took it out, just in case. You're right, there's no need, though I don't think it affects the sysimage. I can't confirm I could access it locally, also I think Markdown doesn't go into the sysimage, only its pkgimage.

Reply:
> I can't confirm I could access it locally

I downloaded a build from https://buildkite.com/julialang/julia-master/builds/39840#0191d31d-7916-4484-9b0b-f6c0aed789d1:
```julia-repl
julia> using Markdown

julia> typeof(Markdown.dummy)
Markdown.MD
```